pyproject.toml

```diff
@@ -0,0 +1,64 @@
+[tool.poetry]
+name = "cassio"
+version = "0.1.5"
+description = """A framework-agnostic Python library to seamlessly integrate \
+                 Apache Cassandra(R) with ML/LLM/genAI workloads."""
+authors = [
+    "Stefano Lottini <stefano.lottini@datastax.com>"
+]
+license = "Apache-2.0"
+readme = "README.md"
+packages = [
+    { include = "cassio", from = "src" }
+]
+keywords = ["cassandra", "ai", "llm", "genai", "astradb"]
+homepage = "https://cassio.org"
+repository = "https://github.com/hemidactylus/cassio"
+classifiers = [
+    "Development Status :: 4 - Beta",
+    "Intended Audience :: Developers",
+    "License :: OSI Approved :: Apache Software License",
+    "Programming Language :: Python :: 3",
+    "Programming Language :: Python :: 3.8",
+    "Programming Language :: Python :: 3.9",
+    "Programming Language :: Python :: 3.10",
+    "Programming Language :: Python :: 3.11",
+    "Programming Language :: Python :: 3.12",
+    "Programming Language :: Python :: 3 :: Only",
+    "Topic :: Scientific/Engineering",
+    "Topic :: Scientific/Engineering :: Artificial Intelligence",
+]
+
+[tool.poetry.dependencies]
+python = ">=3.8,<4.0"
+cassandra-driver = "^3.28.0"
+requests = "^2.31.0"
+numpy = ">=1.0"
+
+[tool.poetry.group.dev.dependencies]
+black = "~23.11.0"
+ruff = "~0.1.5"
+mypy = "~1.7.0"
+pytest-asyncio = "~0.23.2"
+pytest = "~7.4.3"
+
+[build-system]
+requires = ["poetry-core>=1.0.0"]
+build-backend = "poetry.core.masonry.api"
+
+[tool.mypy]
+#disallow_any_generics = true
```

Comment:
The mypy.ini wasn't used...
So if we want to use a custom config, there's a bunch of mypy errors to fix first 😬

Reply:
Oooh. Very interesting, that makes for some nice future work. Challenge accepted :)